### DISCUSSION TITLE
I need help, I received an email saying that my identity could not be verified.

### DISCUSSION THREAD

**Discussion by Camilog03** [community/community#119005] [GitHub Education]
Help, plz

So hi, I need help I was make my request to use GitHub for students and I send all my documents but when I did it, I didn´t make de A2F so for that I do it again before and  that time i received a email saying it can´t be confirmed my documents and so i try it again but in the page it appears like I´m approved. and i dont know if i get in or not
![image](https://github.com/community/community/assets/165982991/9d27b87b-644f-424a-92bf-8c989e95f807)
![image](https://github.com/community/community/assets/165982991/a0fe24fe-86d1-4944-8777-b159b40457a6)

**Comment by samus-aran** [comment]
Hi @iBlacKings 

Looks like that email was sent in error 👀 The top most status on your application page is accurate. 👍 

I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
